src/PersonObjects/PersonMethods.ts

```diff
@@ -104,6 +108,9 @@ export function gainIntelligenceExp(this: Person, exp: number): void {
     this.skills.intelligence = Math.floor(this.calculateSkill(this.exp.intelligence, 1));
   }
 }
+
+// Increases all skills' experience based on the given WorkStats and updates the corresponding skill levels.
```

Comment:
Similar comment about using JSDoc (which is mostly just a matter of changing the comment type)